target/linux/generic/backport-5.15/850-v5.17-0009-PCI-aardvark-Assert-PERST-when-unbinding-driver.patch

```diff
@@ -20,7 +20,7 @@ Signed-off-by: Lorenzo Pieralisi <lorenzo.pieralisi@arm.com>
 
 --- a/drivers/pci/controller/pci-aardvark.c
 +++ b/drivers/pci/controller/pci-aardvark.c
-@@ -1721,6 +1721,10 @@ static int advk_pcie_remove(struct platf
+@@ -1919,6 +1919,10 @@ static int advk_pcie_remove(struct platf
```

Comment:
Please remove this patch, it was already applied upstream, see https://github.com/gregkh/linux/commit/246a36b6637740f1be9c3cdd91f022b24b7402e7